### PR TITLE
bygg: package-server option: Split up into deploy-packages and deploy-images and fix remote paths

### PR DESCRIPTION
**Pull request by albsod**

### CODE REVIEW

File: scripts/bygg [hunk 1]
```diff
@@ -336,19 +339,45 @@ sync_packages()
...
 
   rsync -avr "${1}/build*/tmp/deploy/ipk/" "${2}/$packages_path" || return 1
-  rsync -avr "${1}/build*/tmp/deploy/images/" "${2}/builds/${BUILD_TAG}" || return 1
 
   return 0
 }
 
+sync_images()
+{
+  local -r release_codename=$(basename "$MANIFEST_FILE" .xml)
```
Comment (RikardO-HM) [inline scripts/bygg:354]:
```suggestion
```

File: scripts/bygg [hunk 2]
```diff
@@ -383,18 +413,27 @@ if [[ "$RUNNING_IN_DOCKER" != "1" ]]; then
...
   done
 
-  if [[ -n "$package_server" ]]; then
+  if [[ -n "$package_server" ]]; then 
     sync_packages "$YOCTO_FOLDER" "$package_server"
+
     if [ $? -ne 0 ]; then
-        echo "🔴 sync_packages for package_server (${package_server}) failed"
-        exit 1
+      echo "🔴 sync_packages for package_server (${package_server}) failed"
```
Comment (RikardO-HM) [inline scripts/bygg:424]:
```suggestion
      echo "🔴 sync_packages for deploy-packages (${package_server}) failed"
```

File: scripts/bygg (hunk 1 above)
Comment (albsod) [reply scripts/bygg]: fixed

File: scripts/bygg (hunk 2 above)
Comment (albsod) [reply scripts/bygg]: Thanks, fixed

File: scripts/bygg [hunk 3]
```diff
@@ -383,18 +413,27 @@ if [[ "$RUNNING_IN_DOCKER" != "1" ]]; then
...
+      exit 1
     fi
-fi
+  fi
+
+  if [[ -n "$image_server" ]]; then 
+    sync_images "$YOCTO_FOLDER" "$image_server"
+
+    if [ $? -ne 0 ]; then
+      echo "🔴 sync_images for image_server (${image_server}) failed"
```
[the remark this answers is not in the document]
Comment (albsod) [reply scripts/bygg]: Fixed

**Review by RikardO-HM — APPROVED** [review]

**Pull request merged**